### DISCUSSION TITLE
Sqlite for small home self-hosted server?

### DISCUSSION THREAD

**Discussion by git-78** [simple-login/app#1666] [Q&A]
Hi
Can Sqlite be used instead of postgresql for a small home self-hosted server? Is it technically compatible (compatible sql queries used)? I think I saw that developers uses Sqlite to test simple-login on their own computer.

Note: I know that sqlite lacks several features of mainstream DB servers (poor locking, etc.), but for a 1-2 user server, I hope these lacks should be fine

**Comment by nguyenkims** [comment]
@git-78 we haven't tested sqlite and can't say. We do use some specific postgres textsearch feature https://www.postgresql.org/docs/current/datatype-textsearch.html
